charts/osdfir-infrastructure/Chart.yaml

```diff
@@ -32,7 +32,6 @@ sources:
 - https://github.com/google/osdfir-infrastructure
 - https://github.com/google/turbinia
 - https://github.com/google/timesketch
-appVersion: "0.1.0"
```

Comment:
you only remove this line, is not needed anymore or do you want to up this to 0.2.0 as well?

Reply:
Not needed anymore as the `appVersion` is used for the docker container tag (if we didn't set one already). For OSDFIR Infra sakes, since bundling multiple tools providing it wouldn't do anything/incorrectly set the tag